docs/common/headingManager.js

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+
+import * as Utilities from '~/common/utilities';
+
+/**
+ * These types directly correspond to MDAST node types
+ */
+export const HeadingType = {
+  Text: 'text',
+  Code: 'inlineCode',
```

Comment:
```suggestion
  InlineCode: 'inlineCode',
```
Why not keep the original name here? 🤔